inc/providers/class-wp-sitemaps-users.php

```diff
@@ -37,7 +37,31 @@ public function __construct() {
 	 * @return array $url_list Array of URLs for a sitemap.
 	 */
 	public function get_url_list( $page_num, $object_subtype = '' ) {
-		$query    = $this->get_public_post_authors_query( $page_num );
+		/**
+		 * Filters the users URL list before it is generated.
+		 *
+		 * Passing a non-null value will effectively short-circuit the generation,
+		 * returning that value instead.
+		 *
+		 * @since 5.5.0
+		 *
+		 * @param array  $url_list The URL list. Default null.
+		 * @param int    $page_num Page of results.
+		 */
+		$url_list = apply_filters(
+			'wp_sitemaps_taxonomies_pre_url_list',
```

Comment:
should be `wp_sitemaps_users_pre_url_list`

Reply:
Whoops! Thanks